tools/patchwork_integration_test.sh

```diff
@@ -0,0 +1,102 @@
+#! /bin/bash
+set -e
+HOME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+PUBIN_PATH=$1
+SINCE=$2
```

Comment:
I can't find a documentation of cmdline parameters. You wrote I shall run './patchwork_integration_test.sh'. That fails of course. Please document how to invoke the script.

Reply:
I documented it in the commit message. Will add some usage documentation inside the script too